src/mlpack/tests/CMakeLists.txt

```
@@ -87,7 +87,7 @@ add_executable(mlpack_test
   main_tests/gmm_probability_test.cpp
   main_tests/gmm_train_test.cpp
   main_tests/hmm_generate_test.cpp
-  main_tests/hmm_loglik_test.cpp
+  main_tests/hmm_loglik_test.cpp 
```

```suggestion
  main_tests/hmm_loglik_test.cpp
```
No need for the extra space. :)